templates/403.html

```diff
@@ -0,0 +1,7 @@
+{% extends "error.html" %}
+
+{% block error_detail %}
+<p>Access to 4DN Jupyterhub Denied! </p>
+<p>To help combat abuse, the 4DN Juptyerhub now requires a lab assocation in order to access the hub.
+    Double check that you have been granted a lab association.</p>
+{% endblock %}
```

Comment:
Do individual users know how to do this? It might be you want to be more vague, but I don't feel strongly about this.
```suggestion
<p>Access to 4DN Jupyterhub Denied! </p>
<p>To help combat abuse, the 4DN Juptyerhub is only available to users with a lab association.
    If your work <i>is</i> associated with a lab, you may be improperly configured.</p>
{% endblock %}
```